app/vite.config.ts

```diff
@@ -15,6 +15,9 @@ export default defineConfig({
   },
 
   plugins: [
+    replace({
+      'svelte-i18n-lingui': 'svelte-icu-l10n',
```

Comment:
You don't need this replacement.
* You can remove `T` component and replace it with simple `{$t...}`
* Where `plural()` is used, import it from our l10n.ts.